string nome_studente
uint8 eta
string corso
string testo_concat
